plugins/woocommerce/client/blocks/bin/webpack-config-interactive-blocks.js

```diff
@@ -30,6 +30,10 @@ const entries = {
 	// Experimental mini cart frontend modules, only enqueued when experimental-iapi-mini-cart feature flag is enabled.
 	'woocommerce/mini-cart': './assets/js/blocks/mini-cart/iapi-frontend.ts',
 
+	// Product elements frontend module. Share by several blocks.
+	'woocommerce/product-element':
```

Comment:
I'd keep it plural for consistency

```suggestion
	'woocommerce/product-elements':
```

Reply:
Makes sense! Done in 71a9066747bd272f46bb4513ad94628329de257e.